pages/common/cs.md

```diff
@@ -0,0 +1,29 @@
+# coursier
+
+> Application and artifact manager for the Scala language, it can install Scala applications and setup your Scala development environment.
+> Some subcommands such as `install`, `launch`, `java`, `fetch`, `resolve`, `complete-dep`, etc. have their own usage documentation, accessible via `tldr cs subcommand`.
+> More information: <https://get-coursier.io/docs/overview>.
+
+- Show the current version:
+
+`cs version`
```

Comment:
```suggestion
- Display version:

`cs version`
```

Optional fix: broken description: `Display` expected.
Optional fix: broken position: example can be moved to the bottom.